### DISCUSSION TITLE
Selected answers appear twice

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#51842] [Mobile] [answered]
GitHub Mobile beta

Every discussion I visit that has a selected answer shows the answer twice. This is a regression. 
![Screenshot_20230403-090837.png](https://user-images.githubusercontent.com/2119212/229519263-74eca4b0-4fde-453c-aaf4-b587568c41a3.png)

**Comment by jsoref** [comment]
It's possible one is the thing attached to the question and the other is the answer in context. The experience for that in a browser isn't as confusing as I'm seeing here.

**Comment by iskae** [comment, marked as answer]
Hello @jsoref 👋 

Thanks so much for reaching out!

This is an expected change so that the users are able to see the selected answers in the discussion headers and does not need to scroll through suggested answers to find the actual answer. We still keep the answer header in the list too so that we can differentiate between a suggested answer and the comment that has been selected as an answer 👍 

Happy to forward your concern to our designers though 🙇 Let's see if we can improve the experience in here 😊 

Cheers!
